### ISSUE TITLE
Crashes On Launching Second Time

### ISSUE THREAD

**Issue by hackingguy** [files-community/Files#4558]
**Explain the bug**
On Launching App second time, got a crash, on first launch just changed some settings and everything was working perfectly fine.

**To reproduce**
1. After first launch, Everything was working fine, I just changed theme to dark mode and exited.
2. But on next launch I was on a screen like in below screenshot and got a immediate crash
3. Logs are provided below

**Expected behavior**
App should start normally.

**Logfile**
[debug_fulltrust.txt](https://github.com/files-community/Files/files/6292588/debug_fulltrust.txt)
[debug.txt](https://github.com/files-community/Files/files/6292590/debug.txt)

```
2021-04-12 00:50:37.4360|ERROR|.ctor|The RPC server is unavailable. (Exception from HRESULT: 0x800706BA)|System.Exception: The RPC server is unavailable. (Exception from HRESULT: 0x800706BA)
   at System.Runtime.InteropServices.McgMarshal.ThrowOnExternalCallFailed(Int32, RuntimeTypeHandle) + 0x21
   at __Interop.ComCallHelpers.Call(__ComObject, RuntimeTypeHandle, Int32, Void*) + 0xbe
   at __Interop.ForwardComStubs.Stub_15[TThis](__ComObject, Int32) + 0x2e
   at Files!<BaseAddress>+0x1d4f9b8
   at System.Runtime.InteropServices.WindowsRuntime.IIterator_PrivateRCWAdapter`1.MoveNext() + 0x104
   at Files.UserControls.Widgets.RecentFiles.<PopulateRecentsList>d__17.MoveNext() + 0x1ec
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__7_0(Object) + 0x1e
   at System.Action`1.Invoke(T) + 0x28
   at System.Threading.WinRTSynchronizationContext.Invoker.InvokeCore() + 0x33
   at System.Runtime.InteropServices.McgMarshal.ThrowOnExternalCallFailed(Int32, RuntimeTypeHandle) + 0x21
   at __Interop.ComCallHelpers.Call(__ComObject, RuntimeTypeHandle, Int32) + 0xb8
   at __Interop.ForwardComStubs.Stub_19[TThis](__ComObject, Int32) + 0x24
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object, UnhandledErrorDetectedEventArgs) + 0x3d
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object, UnhandledErrorDetectedEventArgs) + 0x75
   at System.EventHandler`1.Invoke(Object, TEventArgs) + 0x2e
   at __Interop.Intrinsics.HasThisCall__24[TArg0](Object, IntPtr, Object, TArg0) + 0x36
   at Files!<BaseAddress>+0x158d6e0
   ```

**Screenshots**
![RAoWJsxhvq](https://user-images.githubusercontent.com/28388650/114318112-244e2e80-9b29-11eb-8222-91b7699b9bc3.png)

**System Information**
 - OS Version: 19041.867
 - App version: 1.5

**Additional comment**
N/A

**Comment by duke7553** [comment]
@gave92 Just wanted to check if this one was still relevant after the work done to the FullTrustProcess

**Comment by gave92** [comment]
I haven't seen this in a long time but the stacktrace seems to be pointing at PopulateRecentsList() which is not related to FTP.

**Comment by gave92** [comment]
Ugh just noticed this PopulateRecentsList() which is unchanged from 1.5.. I'll fix it.

```cs
try
{
	PopulateRecentsList();
}
catch (Exception ex) <- DOES NOT WORK WITH ASYNC VOID METHODS!!
{
	App.Logger.Info(ex, "Could not fetch recent items");
}

...
private *async void* PopulateRecentsList()

```

**Comment by yaichenbaum** [comment]
This issue should be resolved with #5629.
